dev/tasks/verify-rc/github.linux.amd64.yml

```diff
@@ -65,6 +65,14 @@ jobs:
           TEST_DEFAULT: 0
           TEST_{{ target|upper }}: 1
         {% if use_conda %}
```

Comment:
The job uses this template: https://github.com/apache/arrow/blob/main/dev/tasks/verify-rc/github.linux.amd64.docker.yml instead

Reply:
Oh, sorry!